src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests.vb

```diff
@@ -352,5 +352,106 @@ BC30581: 'AddressOf' expression cannot be converted to 'Integer' because 'Intege
     IObjectCreationExpression (Constructor: Sub System.Guid..ctor()) (OperationKind.ObjectCreationExpression, Type: System.Guid) (Syntax: 'New System.Guid()')
     IOperation:  (OperationKind.None) (Syntax: 'AddressOf Main')")
         End Sub
+
+        <Fact>
+        <WorkItem(19819, "https://github.com/dotnet/roslyn/issues/19819")>
+        Public Sub UsingDeclarationSyntaxNotNull()
```

Comment:
>UsingDeclarationSyntaxNotNull [](start = 19, length = 29)

Please assert diagnostics in all new tests, we should see if there is anything wrong with the code and what exactly is wrong.